proto/tendermint/farming/v1beta1/farming.proto

```diff
@@ -22,9 +22,10 @@ message Params {
     (gogoproto.nullable)     = false
   ];
 
-  // The universal epoch length in number of days. Every process for staking and
-  // reward distribution is executed with this epoch_days frequency
-  uint32 epoch_days = 2 [(gogoproto.moretags) = "yaml:\"epoch_days\""];
+  // next_epoch_days is the epoch length in number of days
+  // it updates internal parameter called CurrentEpochDays that is used to process
```

Comment:
Maybe we can use the term **internal state** instead of **internal parameter** since that is actually not a parameter.

Reply:
You're right. I'll make that change.